website/notifications/model.py

```diff
@@ -4,18 +4,27 @@
 from modularodm.exceptions import ValidationValueError
 
 from website.project.model import Node
-from website.notifications.constants import NOTIFICATION_TYPES
+from website.notifications.constants import NOTIFICATION_TYPES, NODE_SUBSCRIPTIONS_AVAILABLE, USER_SUBSCRIPTIONS_AVAILABLE
+import re
 
 
 def validate_subscription_type(value):
     if value not in NOTIFICATION_TYPES:
         raise ValidationValueError
 
 
+def validate_event_type(value):
+    prefix = re.compile('[[a-zA-Z0-9]*_]?')
+    if prefix.search(value):
+        value = 'file_updated'
```

Comment:
Could you describe what's happening here? This seems like there's something nonstandard being handled.